src/content/en/showcase/2016/service-worker-perf.md

```diff
@@ -0,0 +1,679 @@
+
+The [Google I/O web app](https://events.google.com/io2016/) (IOWA for short) is a [progressive web app](/web/progressive-web-apps/) that leveraged most of the new capabilities offered by service workers to deliver a rich, app-like experience to its users. It also used Google Analytics to capture key performance data and usage patterns from its large and diverse user audience.
+
+This case study explores how IOWA used Google Analytics to answer key performance questions and report on the real-world impact of service workers.
+
+## Starting with the questions
+
+Any time you implement analytics in a website or application, it's important to start by identifying the questions you're trying to answer from the data you'll be  collecting.
+
+While we had several questions we wanted to answer, for the purposes of this case study, let's focus on two of the more interesting ones.
+
+**1. Is service worker caching more performant than the existing HTTP caching mechanisms available in all browsers?**
+
+We already expect pages to load faster for returning visitors than for new visitors since browsers can cache requests and serve them instantly on repeat visits.
+
+Service workers offers alternative caching capabilities that give developers fine-grained control over exactly what and how caching is done. In IOWA, we optimized our service worker implementation so that every asset would be cached, so returning visitors could use the app completely offline.
+
+But would this effort end up being any better than what the browser already does by default? And if so, how much better?&nbsp;<sup><a href="#footnotes">[1]</a></sup>
+
+**2. How does service worker impact the experience of the site loading?**
+
+In other words, how fast does it *feel* like the site is loading, regardless of the actual load times as measured by traditional page load metrics?
+
+Answering questions about how an experience feels is obviously not an easy task, and no metric is going to perfectly represent such a subjective sentiment. That being said, there are definitely some metrics that are better than others, so choosing the right ones is important.
+
+### Choosing the right metric
+
+Google Analytics, by default, tracks [page load times](https://support.google.com/analytics/answer/1205784) (via the [Navigation Timing API](https://developer.mozilla.org/en-US/docs/Web/API/Navigation_timing_API)) for 1% of a site's visitors, and it makes that data available via metrics like [Avg. Page Load Time](/analytics/devguides/reporting/core/dimsmets#view=detail&group=site_speed&jump=ga_avgpageloadtime).
```

Comment:
The actual metric name in Google Analytics is literally "Avg. Page Load Time (sec)". I've removed the "(sec)" part for brevity, but I think it makes sense to refer to it by the actual name to avoid confusion.

In addition, some of the GA autocomplete tools won't show this metric if the user types in "Average" since it's not a literal match. It's caused confusion in the past, so I think it's best to use the actual term name.